lib/fastlane/plugin/sentry/actions/sentry_debug_files_upload.rb

```diff
@@ -0,0 +1,166 @@
+module Fastlane
+  module Actions
+    class SentryDebugFilesUploadAction < Action
+      def self.run(params)
+        require 'shellwords'
+
+        Helper::SentryConfig.parse_api_params(params)
+
+        paths = params[:path]
+        paths = ['.'] if paths.nil?
+
+        command = [
+          "debug-files",
+          "upload"
+        ]
+        command += paths
+
+        command.push('--type').push(params[:type]) unless params[:type].nil?
+        command.push('--no-unwind') unless params[:no_unwind].nil?
+        command.push('--no-debug') unless params[:no_debug].nil?
+        command.push('--no-sources') unless params[:no_sources].nil?
+        command.push('--ids').push(params[:ids]) unless params[:ids].nil?
+        command.push('--require-all') unless params[:require_all].nil?
+        command.push('--symbol-maps').push(params[:symbol_maps]) unless params[:symbol_maps].nil?
+        command.push('--derived-data') unless params[:derived_data].nil?
+        command.push('--no-zips') unless params[:no_zips].nil?
+        command.push('--info-plist').push(params[:info_plist]) unless params[:info_plist].nil?
+        command.push('--no-reprocessing') unless params[:no_reprocessing].nil?
+        command.push('--force-foreground') unless params[:force_foreground].nil?
+        command.push('--include-sources') unless params[:include_sources] != true
+        command.push('--wait') unless params[:wait].nil?
+        command.push('--upload-symbol-maps') unless params[:upload_symbol_maps].nil?
```

Comment:
`m`: That is the same as https://github.com/getsentry/sentry-fastlane-plugin/blob/1b5318519a27911f845b3e3c2949dac25943c90c/lib/fastlane/plugin/sentry/actions/sentry_upload_dif.rb#L17-L31

It would be great, if we could reuse some code.